Source/Core/Logging/CorrelationIDGenerator.swift

```diff
@@ -1,26 +1,28 @@
 import Foundation
 
-protocol CorrelationIDGenerating {
+protocol CorrelationIDManaging {
     
     func generateCorrelationID() -> String
-    
+    func destroyCorrelationID()
 }
 
-final class CorrelationIDGenerator: CorrelationIDGenerating {
-    
-    private let createUUID: () -> UUID
-    
-    // MARK: - Init
+final class CorrelationIDManager: CorrelationIDManaging {
```

Comment:
What are the general expectations from this Pull Request in terms of reviewing? Should we look at finding things that may be wrong, or has everything been individually reviewed already and we just need to ensure automation passes, then approve and let in?

I tried looking through and the changes are so complex I wouldn't know what you want me to do here ❓

Reply:
No review expected. PR only exists to update branch